src/main/scala/org/hammerlab/guacamole/assembly/DeBruijnGraph.scala

```diff
@@ -293,9 +304,9 @@ object DeBruijnGraph {
     graph
   }
 
-  def mergeKmers(kmers: Seq[Kmer]): Sequence = {
-    val head = kmers.headOption.map(h => h.take(h.length - 1)).getOrElse(Seq.empty)
-    val rest = kmers.map(_.last)
+  def mergeKmers(kmers: Seq[Kmer], kmerSize: Int): Sequence = {
+    val head = kmers.headOption.getOrElse(Seq.empty)
+    val rest = kmers.tail.flatMap(kmer => kmer.takeRight(kmer.length - kmerSize + 1))
```

Comment:
Yea, I think that makes sense. I think I will change this to 'mergeOverlaping` or something of that sort and specify an overlap size?